packages/x-components/src/x-modules/search/wiring.ts

```diff
@@ -84,6 +84,13 @@ export const setSort = wireCommit('setSort');
  */
 export const setPage = wireCommit('setPage');
 
+/**
+ * Sets the search state `page`.
```

Comment:
Correct this doc.

```suggestion
 * Sets the search state `params`.
```

- [ ] Correct doc